{
  LOG.info("exception executing " + getName() + ", aborting transaction");
  try {
    objectManager.abortTransaction();
  }
 catch (  Exception e2) {
    LOG.error("failure during abort",e2);
  }
}
